sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/SarifParserCallbackImpl.java

```diff
@@ -192,12 +194,14 @@ private NewExternalIssue newExternalIssue(String ruleId) {
   }
 
   private void createIssue(InputFile inputFile, String ruleId, Location primaryLocation, Collection<Location> secondaryLocations, String repositoryKey) {
+    boolean isOwnRepository = isOwnRepository(repositoryKey);
```

Comment:
_minor_ : for clarity, I'd suggest to rename `own`, although I leave it to you if you think it's clear enough
```suggestion
    boolean isSonarsourceRepository = isSonarsourceRepository(repositoryKey);
```